beacon-chain/chaintest/tests/state-tests/block-processing.yaml

```diff
@@ -4,35 +4,35 @@ test_suite: prysm
 fork: sapphire
 version: 1.0
 test_cases:
+#  - config:
+#      epoch_length: 64
+#      deposits_for_chain_start: 5000
+#      num_slots: 32 # Testing advancing state to slot < EpochLength
+#    results:
+#      slot: 9223372036854775840
+#      num_validators: 5000
   - config:
       epoch_length: 64
-      deposits_for_chain_start: 5000
-      num_slots: 32 # Testing advancing state to slot < EpochLength
-    results:
-      slot: 32
-      num_validators: 5000
-  - config:
-      epoch_length: 63
       deposits_for_chain_start: 16384
       num_slots: 64 # Testing advancing state to exactly slot == EpochLength
       deposits:
-        - slot: 1
+        - slot: 9223372036854775809
```

Comment:
haven't spent much time to research, I wonder if there's an elegant way to represent this, yams won't let me do + or power notations

Reply:
id say use a string @terenc3t